### ISSUE TITLE
npm run debug fails on windows

### ISSUE THREAD

**Issue by tivac** [ampproject/worker-dom#3]
Using powershell in windows 10.

```
PS C:\Users\Patrick\Desktop\worker-dom> npm run debug

> @ampproject/worker-dom@0.1.0 predebug C:\Users\Patrick\Desktop\worker-dom
> DEBUG_BUNDLE=true npm run ~rollup

'DEBUG_BUNDLE' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @ampproject/worker-dom@0.1.0 predebug: `DEBUG_BUNDLE=true npm run ~rollup`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @ampproject/worker-dom@0.1.0 predebug script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Patrick\AppData\Roaming\npm-cache\_logs\2018-08-22T05_51_13_496Z-debug.log
```

I think `npm run ~rollup --environment=DEBUG_BUNDLE` would do the same thing in a cross-platform way w/o the need for any other tools.

Looks like the `build` task is gonna have the same problem. 😞

**Comment by kristoferbaxter** [comment]
Likely will leverage the excellent library from Kent Dodds, [cross-env](https://github.com/kentcdodds/cross-env), to fix this issue.

We should also enable testing of windows environments in Travis to prevent regressions.

**Comment by kristoferbaxter** [comment]
These two issues have been resolved with #21. However, we don't have an environment setup yet for testing ongoing changes.

I'll close this issue and add some additional notes for #25.
